ext/handlers/callbackquery.go

```diff
@@ -11,27 +12,31 @@ import (
 type CallbackQuery struct {
 	AllowChannel bool
 	Filter       filters.CallbackQuery
-	Response     Response
```

Comment:
What is the reasoning behind this change?

Reply:
the signature (b *gotgbot.Bot, ctx *ext.Context) error, allowing for greater versatility in defining response behavior.

Reply:
That is the same signature as the existing `Response` type... This is a noop and completely unnecessary change.
The definition is already: `type Response func(b *gotgbot.Bot, ctx *ext.Context) error`